amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H

```diff
@@ -243,7 +243,7 @@ struct InitDataOp<TurbineFast, SrcTrait>
         amrex::ParallelDescriptor::Comm_dup(
             amrex::ParallelDescriptor::Communicator(), tdata.tcomm);
 
-        int sz_info[4]{0, 0, 0, 0};
+        int sz_info[5]{0, 0, 0, 0, 0};
```

Comment:
I would be inclined to make this an `amrex::Vector<int> sz_info(5, 0)`. Also, is there an assert you can add somewhere to check that it was correctly populated? Maybe below on line 255? I would hate for this to happen again.

Reply:
Different type, like a pointer? You would get that with `sz_info.data()`. Then your assert would just be `sz_info.size() == 5`

Reply:
The problem was not in the assert. It is in another fast function where it gets called. 
This is the error
`/home/lmartine/amr-wind-tony/amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:264:52: error: no matching function for call to 'Bcast(amrex::Vector<int>&, int, const int&, MPI_Comm&)'
             sz_info, 3, info.root_proc, tdata.tcomm);`

We need to leave it as `int sz_info[5]{0, 0, 0, 0, 0};`

Reply:
Lets drop this, I have an upcoming PR that will fix all these.